.circleci/config.yml

```diff
@@ -151,9 +187,9 @@ workflows:
   version: 2
   build_all:
     jobs:
+      - build-php73-laravel58
+      - build-php72-laravel58
+      - build-php71-laravel58
       - build-php73-laravel57
       - build-php72-laravel57
       - build-php71-laravel57
-      - build-php73-laravel56
-      - build-php72-laravel56
-      - build-php71-laravel56
```

Comment:
because we don't drop Laravel `5.6` support I would like to keep the lowest possible build here.